spec.bs

```diff
@@ -165,3 +165,9 @@ The {{HTMLFencedFrameElement/src}} IDL attribute must [=reflect=] the respective
 <h3 id=dimension-attributes>Dimension attributes</h3>
 
 This section details monkeypatches to [[!HTML]]'s <a href="https://html.spec.whatwg.org/multipage/embedded-content-other.html#dimension-attributes">Dimension attributes</a> section. This section will be updated to include <{fencedframe}> in the list of elements that the <code>[=width=]</code> and <code>[=height=]</code> dimension attributes apply to.
+
+<h3 id=coep>New fencedframe [=request=] [=request/destination=]</h3>
+
+Update the list of <a href="https://fetch.spec.whatwg.org/#concept-request-destination">associated destinations</a> to include a new entry, <code>fencedframe</code>. It will have the initiator <code>""</code>, the CSP directive <code>child-src</code>, and the features <code>HTML's &lt;fencedframe&gt;</code>.
+
```

Comment:
`<code>child-src</code>` should I think be `<code>fenced-frame-src</code>`. We'll need a monkeypatch section for this as well

Reply:
Would we want both? fenced-frame-src seems similar to the [frame-src directive](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/frame-src), but that's not mentioned anywhere in the [fetch spec](https://fetch.spec.whatwg.org/). And I think if other frame types (like iframes) listen to the child-src directive, it would make sense for fenced frames to listen to it as well.

Reply:
Ah I see. OK maybe just `child-src` here is fine, especially since we don't use frame-src here in the fetch spec. That is a little surprising to me though. I guess we're listing the closest-but-most-general directive that applies.

Reply:
And we should still monkey-patch a fenced-frame-src directive somewhere as part of a different PR?

Reply:
Yes. looks like you're taking care of that in another CL so I'll resolve this.